examples/src/main/java/ai/djl/examples/inference/SentimentAnalysis.java

```diff
@@ -56,6 +57,8 @@ public static Classifications predict()
                 Criteria.builder()
                         .optApplication(Application.NLP.SENTIMENT_ANALYSIS)
                         .setTypes(String.class, Classifications.class)
+                        // This model was traced on CPU and can only run on CPU
```

Comment:
I though it is because RNN bug  in MXNet MKLDNN

Reply:
This is a PT model.